ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -129,7 +135,6 @@ if [[ "$DISABLE_WORKLOAD_MONITORING" == "true" ]]; then
 fi
 
 HYPERSHIFT_SWITCH=""
-SUBNET_ID_SWITCH=""
 if [[ "$HOSTED_CP" == "true" ]]; then
   HYPERSHIFT_SWITCH="--hosted-cp --classic-oidc-config"
```

Comment:
classic-oidc-config is not the suggested option, it is hidden. It still can be used, it depends on you if use the the byo oidc config

Reply:
It is just a workaround. We need to do more around the OIDC config.